modules/object-cache/health/load.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * Module Name: Health Check
+ * Description: Recommend a persistent object cache for sites with non-trivial amounts of data.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+add_filter( 'site_status_tests', 'oc_health_add_tests' );
+add_filter( 'site_status_persistent_object_cache', 'oc_health_should_persistent_object_cache' );
+
+/**
+ * Adds a health check testing for and suggesting a persistent object cache backend.
+ *
+ * @param array $tests An associative array of direct and asynchronous tests.
+ * @return array
+ */
+function oc_health_add_tests( $tests ) {
+	$tests['direct']['persistent_object_cache'] = array(
+		'label' => 'persistent_object_cache',
+		'test'  => 'oc_health_persistent_object_cache',
+	);
+
+	return $tests;
+}
+
+/**
+ * Callback for `persistent_object_cache` health check.
+ *
+ * @return array
+ */
+function oc_health_persistent_object_cache() {
+	/**
+	 * Filter the action URL for the persistent object cache health check.
+	 *
+	 * @param string $action_url Learn more link for persistent object cache health check.
+	 */
+	$action_url = apply_filters(
+		'site_status_persistent_object_cache_url',
+		/* translators: Localized Support reference. */
+		__( 'https://wordpress.org/support/article/optimization/#object-caching', 'performance-lab' )
```

Comment:
There is no `object-caching` section on the Optimization page. Is it yet to be added? :thinking: If not, then we probably need to update it to something like `https://wordpress.org/support/article/optimization/#server-side-caching`?

Reply:
Yeah, it's mentioned in the PR description as a TODO.